static/app/types/chromeTrace.d.ts

```diff
@@ -23,7 +23,7 @@ declare namespace ChromeTrace {
     samples: ReadonlyArray<any>;
   }
 
-  type ArrayFormat = ReadonlyArray<Event>;
+  type ArrayFormat = Array<Event | ProfileEvent>;
```

Comment:
What about all the other event types you defined below? Also, the `ph` for `Event` already defines `P` should this be updated?